index.js

```diff
@@ -0,0 +1,45 @@
+import express from "express";
+import geohash from "ngeohash";
+import {
+  uniqueNamesGenerator,
+  adjectives,
+  colors,
+  animals,
+} from "unique-names-generator";
+
+/**
+ * @type {Object.<string, Set>}
+ */
+const grids = {};
+const app = express();
+
+app.use(express.json());
+app.use(express.static("public"));
+
+app.post("/api/initial", (req, res) => {
+  const { lat, lon } = req.body;
+  // geohash of length 5 is <= 4.89km X 4.89km
+  const hash = geohash.encode(lat, lon).slice(0, 5);
```

Comment:
Use the precision param instead of manually slicing
https://www.npmjs.com/package/ngeohash#geohashencode-latitude-longitude-precision9